reana_cluster/backends/kubernetes/templates/persistentvolumeclaims/ceph.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   - ReadWriteMany
   resources:
     requests:
-      storage: 10Gi
+      storage: 200Gi
```

Comment:
Sample value, should be set at cluster init with `reana-cluster --cephfs 200 init`